lib/templates/crowdin.yml.tpl

```diff
@@ -0,0 +1,38 @@
+# THIS FILE IS AUTOMATICALLY GENERATED. DO NOT EDIT MANUALLY
+
+commit_message: 'feat: new translations'
+files:
+  # Content generate for version: "current"
+  - source: /content/en-US/docs/*.md
+    translation: /content/%locale%/docs/%original_file_name%
+  - source: /content/en-US/docs/development/*.md
+    translation: /content/%locale%/docs/development/%original_file_name%
+  - source: /content/en-US/docs/tutorial/*.md
+    translation: /content/%locale%/docs/tutorial/%original_file_name%
+  - source: /content/en-US/website/*.yml
+    translation: /content/%locale%/website/%original_file_name%
+  - source: /content/en-US/docs/api/*.md
+    translation: /content/%locale%/docs/api/%original_file_name%
+  - source: /content/en-US/docs/api/structures/*.md
+    translation: /content/%locale%/docs/api/structures/%original_file_name%
+  - source: /content/en-US/website/blog/*.md
+    translation: /content/%locale%/website/blog/%original_file_name%
+
```

Comment:
Why not always add `current` to `versions` when calling `generateCrowdinConfig`, so this isn't needed? It'll make it less error-prone to edit this file.

```suggestion
```